MSAL/test/unit/native_auth/network/MSALNativeAuthRequestErrorHandlerTests.swift

```diff
@@ -258,7 +258,50 @@ class MSALNativeAuthResponseErrorHandlerTests: XCTestCase {
         wait(for: [expectation], timeout: 1)
     }
 
-    func test_shouldFailWithDecodeError_whenStatusCode400AndJSONMissing() throws {
+    func test_shouldDecodeErrorWithCorrectResponseSerializer_whenStatusCode400AndInvalidCode() throws {
```

Comment:
This validates that any `invalid code` allows for the object as a whole to be decoded but the Type will be `.none`